plugin/src/test/groovy/com/novoda/test/TestProject.groovy

```diff
@@ -129,7 +133,7 @@ ${project.additionalConfiguration}
     }
 
     protected static String formatPlugins(TestProject project) {
-        "${project.plugins.join(',\n')}"
```

Comment:
was this working at all before? 💃

Reply:
tests were passing :)
I think it was working because we never put more than 1 plugin. 😄

Reply:
exactly ^^